src/app/src/main/resources/metadata/input-time-series-config-jsonata.txt

```diff
@@ -1,75 +1,94 @@
 (
 $areaNames := $distinct(data.area_name);
+$lineColor := "rgb(51, 51, 51)";
+$highlightColor := "rgb(255, 51, 51)";
+$subplotHeight := 1/subplots.rows * 0.6;
```

Comment:
Calculate subplot height as a fraction of total plot height - based on number of rows, then * 0.6 to provide spacing.